cargo-binstall 0.13.2 failed to install cargo-watch

cargo-binstall 0.13.2 failed to install cargo-watch with the following error:

```
05:14:15 [INFO] Resolving package: 'cargo-watch'
05:14:16 [INFO] The package will be downloaded from github.com
05:14:16 [INFO] This will install the following binaries:
05:14:16 [INFO]   - cargo-watch (cargo-watch -> /home/runner/.cargo/bin/cargo-watch-v8.1.2)
05:14:16 [INFO]   - cargo-watch (cargo-watch -> /home/runner/.cargo/bin/cargo-watch-v8.1.2)
05:14:16 [INFO] And create (or update) the following symlinks:
05:14:16 [INFO]   - cargo-watch (/home/runner/.cargo/bin/cargo-watch -> cargo-watch-v8.1.2)
05:14:16 [INFO]   - cargo-watch (/home/runner/.cargo/bin/cargo-watch -> cargo-watch-v8.1.2)
05:14:16 [INFO] Installing binaries...
Error: 05:14:16 [ERROR] Fatal error:
binstall::binfile

  × bin file /home/runner/.cargo/bin/cargo-binstallDqdKr7/bin-cargo-watch-
  │ x86_64-unknown-linux-gnu-GhCrateMeta/cargo-watch-v8.1.2-x86_64-unknown-
  │ linux-gnu/cargo-watch not found
```

full logs:
cargo-binstall 0.13.1 (passed): https://github.com/taiki-e/install-action/actions/runs/3117118429/jobs/5055540606
cargo-binstall 0.13.2 (failed): https://github.com/taiki-e/install-action/actions/runs/3117122909/jobs/5055549681

This was originally found in https://github.com/taiki-e/install-action/issues/21, but I can confirm that this problem occurs locally as well.

--- a/.github/scripts/cargo-tomls/bitbucket-test-Cargo.toml
+++ b/.github/scripts/cargo-tomls/bitbucket-test-Cargo.toml
@@ -13,3 +13,4 @@ bin-dir = "{ bin }{ binary-ext }"
 
 [[bin]]
 name = "cargo-binstall"
+path = "src/main.rs"

--- a/.github/scripts/cargo-tomls/github-test-Cargo.toml
+++ b/.github/scripts/cargo-tomls/github-test-Cargo.toml
@@ -1,8 +1,8 @@
 [package]
 name = "cargo-binstall"
 description = "Rust binary package installer for CI integration"
-repository = "https://gitlab.kitware.com/NobodyXu/hello-world"
-version = "0.2.0"
+repository = "https://github.com/cargo-bins/cargo-binstall"
+version = "0.12.0"
 rust-version = "1.61.0"
 authors = ["ryan <ryan@kurte.nz>"]
 edition = "2021"
@@ -13,3 +13,4 @@ bin-dir = "{ bin }{ binary-ext }"
 
 [[bin]]
 name = "cargo-binstall"
+path = "src/main.rs"

--- a/.github/scripts/cargo-tomls/gitlab-test-Cargo.toml
+++ b/.github/scripts/cargo-tomls/gitlab-test-Cargo.toml
@@ -1,8 +1,8 @@
 [package]
 name = "cargo-binstall"
 description = "Rust binary package installer for CI integration"
-repository = "https://github.com/cargo-bins/cargo-binstall"
-version = "0.12.0"
+repository = "https://gitlab.kitware.com/NobodyXu/hello-world"
+version = "0.2.0"
 rust-version = "1.61.0"
 authors = ["ryan <ryan@kurte.nz>"]
 edition = "2021"
@@ -13,3 +13,4 @@ bin-dir = "{ bin }{ binary-ext }"
 
 [[bin]]
 name = "cargo-binstall"
+path = "src/main.rs"

--- a/.github/scripts/tests.sh
+++ b/.github/scripts/tests.sh
@@ -7,12 +7,13 @@ unset CARGO_HOME
 
 # Install binaries using cargo-binstall
 # shellcheck disable=SC2086
-"./$1" binstall --log-level debug --no-confirm b3sum cargo-binstall
+"./$1" binstall --log-level debug --no-confirm b3sum cargo-binstall cargo-watch
 
 # Test that the installed binaries can be run
 b3sum --version
 cargo-binstall --help >/dev/null
 cargo binstall --help >/dev/null
+cargo watch -V
 
 test_resources=".github/scripts"
 

--- a/.github/scripts/tests.sh
+++ b/.github/scripts/tests.sh
@@ -15,7 +15,7 @@ cargo-binstall --help >/dev/null
 cargo binstall --help >/dev/null
 cargo watch -V
 
-test_resources=".github/scripts"
+test_resources=".github/scripts/cargo-tomls"
 
 # Install binaries using `--manifest-path`
 "./$1" binstall --force --log-level debug --manifest-path "$test_resources/gitlab-test-Cargo.toml" --no-confirm cargo-binstall

--- a/crates/binstalk/Cargo.toml
+++ b/crates/binstalk/Cargo.toml
@@ -13,7 +13,7 @@ license = "GPL-3.0"
 async-trait = "0.1.57"
 bytes = "1.2.1"
 bzip2 = "0.4.3"
-cargo_toml = "0.12.0"
+cargo_toml = "0.12.1"
 clap = { version = "3.2.22", features = ["derive"] }
 compact_str = { version = "0.6.0", features = ["serde"] }
 crates_io_api = { version = "0.8.1", default-features = false }

